Remote worker node at the Edge (#25537)
openshift/openshift-docs: nodes/edge/nodes-edge-remote-workers.adoc

* *Latency spikes or temporary reduction in throughput*: As with any network, any changes in network conditions between your cluster and the remote worker nodes can negatively impact your cluster. These type of situations are beyond the scope of this documentation.

Note the following limitations when planning a cluster with remote worker nodes:
 
* Remote worker nodes are supported on Bare Metal clusters with user-provisioned infrastructure only. 

* {product-title} does not support remote worker nodes that use a different cloud provider than the on-premise cluster.

* Proxies and firewalls can present additional limitations that are beyond the scope of this document. Refer to the relevant {product-title} documentation for how to address such limitations, such as xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[Configuring your firewall] 

> * Moving workloads from one Kunernetes zone to different Kunernetes zone can be problematic due to system and environment issues, such as a specific type of memory not being available in a different zone.

Reviewer: typo: Kubernetes

:context: nodes-edge-remote-workers
[id="nodes-edge-remote-workers"]
= Using remote worker nodes at the network edge
include::modules/common-attributes.adoc[]

toc::[]

> {product-title} clusters can be configured with nodes located at your network edge, in this topic called _remote worker nodes_. A typical cluster with remote worker nodes combines on-premise master and worker nodes with worker nodes in other locations that connect to the cluster. This topic is intended to provide guidance on best practices for using remote worker nodes and does not contain specific configuration details.

Reviewer 2: ```suggestion
You can configure {product-title} clusters with nodes located at your network edge. In this topic, they are called _remote worker nodes_. A typical cluster with remote worker nodes combines on-premise master and worker nodes with worker nodes in other locations that connect to the cluster. This topic is intended to provide guidance on best practices for using remote worker nodes and does not contain specific configuration details.
```

There are multiple use cases across different industries, such as telecommunications, retail, manufacturing, and government, for using a deployment pattern with remote worker nodes. For example, you can separate and isolate your projects and workloads by combining the remote worker nodes into xref:../../nodes/edge/nodes-edge-remote-workers.adoc#nodes-edge-remote-workers-strategies-zones_nodes-edge-remote-workers[Kubernetes zones].

However, having remote worker nodes can introduce higher latency, intermittent loss of network connectivity, and other issues. Among the challenges in a cluster with remote worker node are:

> * *Network separation*: The {product-title} control plane and the remote worker nodes must be able communicate with each other. Because of the distance between the control plane and the remote worker nodes, network issues could prevent this communication. See xref:../../nodes/edge/nodes-edge-remote-workers.adoc#nodes-edge-remote-workers-network_nodes-edge-remote-workers[About network separation with remote worker nodes] for information on how {product-title} responds to network separation and for methods to diminish the impact to your cluster.

Reviewer 2: If you accept the title edits, update them here, too.

* *Power outage*: Because the control plane and remote worker nodes are in separate locations, a power outage at the remote location or at any point between the two can negatively impact your cluster. See xref:../../nodes/edge/nodes-edge-remote-workers.adoc#nodes-edge-remote-workers-power_nodes-edge-remote-workers[About  power loss on remote worker nodes] for information on how {product-title} responds to a node losing power and for methods to diminish the impact to your cluster.

> * *Latency spikes or temporary reduction in throughput*: As with any network, any changes in network conditions between your cluster and the remote worker nodes can negatively impact your cluster. These type of situations are beyond the scope of this documentation.

Reviewer 2: ```suggestion
* *Latency spikes or temporary reduction in throughput*: As with any network, any changes in network conditions between your cluster and the remote worker nodes can negatively impact your cluster. These types of situations are beyond the scope of this documentation.
```

Note the following limitations when planning a cluster with remote worker nodes:
 

> * Remote worker nodes are supported on Bare Metal clusters with user-provisioned infrastructure only. 

Reviewer 2: ```suggestion
* Remote worker nodes are supported on only bare metal clusters with user-provisioned infrastructure. 
```
